blog.md

```diff
@@ -1,18 +1,18 @@
 # Create your first model with Custom Vision and Python
 
-Welcome to this first article in the AI for Developer serie, in this serie of articles I will share tips and tricks around Azure AI with you. My name is Henk Boelman a Cloud Advocate at Microsoft based in the Netherlands, focusing on AI for developers.
+Welcome to this first article in the AI for Developer series, in this series of articles I will share tips and tricks around Azure AI with you. My name is Henk Boelman, a Cloud Advocate at Microsoft based in the Netherlands, focusing on AI for developers.
 
-For the first article I want to share with you how you can create a classification model using the Custom Vision service with the Python SDK.
+In this first article I want to share with you how you can create a classification model using the Custom Vision service with the Python SDK.
 
 ### Why Python and not the visual interface?
-The answer to that is simple, if you build the training process in code you can version it for instance on Github. Having your code versioned means, you can read back what you have done, work together on it and can run it again if you need to. 
+The answer to that is simple, if you build the training process in code you can version it for instance on Github. Having your code versioned means you can read back what you have done, work ina team on it and run it again if you need to. 
 
 Let’s dive into the code! Before we start, I assume you have [Python 3.6](https://www.python.org/downloads/) installed. 
 
 ### Create resources in Azure
-The first thing you need to do is create a Custom Vision service in Azure. If you don’t have an [Azure subscription](https://azure.microsoft.com/free/) you can get 200$ starting credit for the first month. 
+The first thing you need to do is create an Azure Custom Vision service. If you don’t have an [Azure subscription](https://azure.microsoft.com/free/) you can get $200 credit for the first month. 
```

Comment:
Make sure any Azure landing pages including the free subscription links are tracked with a tracking link